mautic/mautic
# To improve user experience, the 'Send Email' button is disabled for contacts who did not provide an email address.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#...
| Issue(s) addressed                     | Fixes #13729 



## Description








---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new and fill in only their name
3. Save & Close
4. The Send email button is disabled with the tooltip, 'Contact does not have an email address. Thus email sending is disabled.'
